packages/dds/sequence/src/sequence.ts

```diff
@@ -730,7 +730,14 @@ export abstract class SharedSegmentSequence<T extends ISegment>
 	 * {@inheritDoc @fluidframework/shared-object-base#SharedObjectCore.applyStashedOp}
 	 */
 	protected applyStashedOp(content: any): unknown {
-		return this.client.applyStashedOp(parseHandles(content, this.serializer));
+		let metadata;
+		metadata = this.intervalCollections.tryGetStashedOpLocalMetadata(content);
+		if (metadata === undefined) {
+			const parsedContent: IMergeTreeOp = parseHandles(content, this.serializer);
+			metadata = this.client.applyStashedOp(parsedContent);
+		}
```

Comment:
```suggestion
		const parsedContent = parseHandles(content, this.serializer)
		const metadata =
			this.intervalCollections.tryGetStashedOpLocalMetadata(parsedContent )
			?? this.client.applyStashedOp(parsedContent );
```

Reply:
a bit of a simplification, and i think we need to parse handles for interval ops as well